### DISCUSSION TITLE
Welcome to python-terrascript Discussions!

### DISCUSSION THREAD

**Discussion by mjuenema** [starhawking/python-terrascript#143] [General]
## Discussions

This page is meant for general discussions around Python-Terrascript, like

  * Asking questions you’re wondering about.
  * Sharing ideas.
  * Engaging with other community members.

**Comment by ZappedC64** [comment]
How do you specify the disk configuration with terrascript? Specifically:

```
disk {
    label = "disk0"
    size  = 20
  }
```
(From this example:  [https://registry.terraform.io/providers/hashicorp/vsphere/latest/docs](url)

and maybe a little more complicated... this...

```data_disk = {
    disk1 = {
      size_gb                   = 30,
      thin_provisioned          = false,
      data_disk_scsi_controller = 0,
    },
    disk2 = {
      size_gb                   = 70,
      thin_provisioned          = true,
      data_disk_scsi_controller = 1,
      datastore_id              = "datastore-90679"
    }
  }
  scsi_bus_sharing          = "physicalSharing" // The modes are physicalSharing, virtualSharing, and noSharing
  scsi_type                 = "lsilogic" // Other acceptable value "pvscsi"
  scsi_controller           = 0 // This will assign OS disk to controller 0
```
(From this example:  [https://registry.terraform.io/modules/Terraform-VMWare-Modules/vm/vsphere/latest](url)

Thanks!
